Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml

```diff
@@ -0,0 +1,852 @@
+  (CASB). It provides rich visibility, control over data travel, and sophisticated
+  analytics to identify and combat cyber threats across all your cloud services. Use the integration to view and resolve alerts, view activities, view files, and view user accounts.
+display: Microsoft Cloud App Security
+name: MicrosoftCloudAppSecurity
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: |-
+        Multiple alerts matching the specified filters.
+        The alert_id should appear similar to "55af7415f8a0a7a29eef2e1f".
+        Mandatory, unless you use a custom filter.
+      isArray: true
+      name: alert_ids
+      required: false
+      secret: false
+    - default: false
+      description: 'A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the custom_filter parameter from the integration instance configuration. For more information about filter syntax, see the (Microsoft documentation)[https://docs.microsoft.com/en-us/cloud-app-security/api-activities#filters].'
+      isArray: false
+      name: custom_filter
+      required: false
+      secret: false
+    - default: false
+      description: A comment explaining why the alerts are being dismissed.
+      isArray: false
+      name: comment
+      required: false
+      secret: false
+    deprecated: false
+    description: Dismisses multiple alerts (bulk dismiss) that match the specified filters.
+    execution: false
+    name: microsoft-cas-alert-dismiss-bulk
+  - arguments:
+    - default: false
+      description: Skips the specified number of records.
+      isArray: false
+      name: skip
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The maximum number of records to return. Default is 50.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'The severity of the alert. Possible values are: "Low", "Medium", and "High".'
+      isArray: false
+      name: severity
+      predefined:
+      - Low
+      - Medium
+      - High
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'The alert resolution status. Possible values are: "Open", "Dismissed", and "Resolved".'
+      isArray: false
+      name: resolution_status
+      predefined:
+      - Open
+      - Dismissed
+      - Resolved
+      required: false
+      secret: false
+    - default: false
+      description: 'A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the custom_filter parameter from the integration instance configuration. For more information about filter syntax, see the (Microsoft documentation)[https://docs.microsoft.com/en-us/cloud-app-security/api-activities#filters].'
```

Comment:
@kirbles19 
Same thing with this command and the rest of the commands
It’s not "override the custom_filter parameter"
It’s override other filters in this command.